vscode-powershell.build.ps1

```diff
@@ -109,6 +111,30 @@ task Test Build, {
     }
 }
 
+task CheckPreview -If { $script:ExtensionVersion -like "*preview*" } `
+    UpdateReadme, UpdatePackageJson
+
+task UpdateReadme {
+    $newReadmeTop = '# PowerShell Language Support for Visual Studio Code
+
+> ## ATTENTION: This is the PREVIEW version of the PowerShell extension for VSCode which contains features that are being evaluated for stable
```

Comment:
I would maybe also make a note about supporting PS 5.1+

Reply:
added. Although in the future both stable and preview will only support 5.1+